mock-rollout/src/Products/ProductService.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import { IProduct } from './ProductModel';
+
+const url = 'https://mock-rollout-e0dc5-default-rtdb.firebaseio.com/products.json';
```

Comment:
move this url to a folder **constants**, and crreate a file called api-endpoints.ts, give it a more specific name and import it from there.

i.e 

export const ENDPOINTS = {
GET_HARDWARE: ''https://mock-rollout-e0dc5-default-rtdb.firebaseio.com/products.json',
}